docs/automation-actions.md

```diff
@@ -179,6 +182,44 @@ automations:
       - action: close@v1
 ```
 
+#### `http-request` :fontawesome-solid-flask: :fontawesome-solid-crown: :fontawesome-brands-github: 
+
+!!! note "Pro feature, in open beta"
+	This feature is under development.
+	Once released, using this feature will be limited to pro subscriptions.
```

Comment:
Will the entire feature be limited, or portions of it? If the latter is correct, we may want to indicate that people who adopt this won't suddenly lose access to everything unless they pay us at an unspecified point in the future.

That might make people feel more comfortable adopting  this feature now.